pepper-apis/src/main/java/org/broadinstitute/ddp/json/sendgrid/SendGridEventTypeAdapter.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.ddp.json.sendgrid;
+
+import java.io.IOException;
+import java.time.Instant;
+
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonToken;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * Convert UNIX timestamp to Java Instant
+ */
+public class SendGridEventTypeAdapter extends TypeAdapter<Instant> {
```

Comment:
We have a few of these time-related adapters in the `ddp.transformers` package. Can you try putting this in there and generalizing it a bit so we can reuse it? E.g. implement `write()` method and rename to something like `InstantToEpochSecondsAdapter`.